multiqc/modules/fastqc/fastqc.py

```diff
@@ -36,20 +38,22 @@ def __init__(self):
         info="is a quality control tool for high throughput sequence data,"\
         " written by Simon Andrews at the Babraham Institute in Cambridge.")
 
+        # For this module the dict will be indexed by tuple(s_name, read_number)
+        # even if there is only one read and read_number is always None.
         self.fastqc_data = dict()
 
+    def gather(self):
         # Find and parse unzipped FastQC reports
         for f in self.find_log_files('fastqc/data'):
-            s_name = self.clean_s_name(os.path.basename(f['root']), os.path.dirname(f['root']))
-            self.parse_fastqc_report(f['f'], s_name, f)
+            self.parse_fastqc_report(f['f'], f=f) #fffffff
 
         # Find and parse zipped FastQC reports
         for f in self.find_log_files('fastqc/zip', filecontents=False):
-            s_name = f['fn']
-            if s_name.endswith('_fastqc.zip'):
-                s_name = s_name[:-11]
+            # Do we still need this??
```

Comment:
We could add `_fastqc.zip` to `config.fn_clean_exts` and then it wouldn't be necessary..